var/spack/repos/spack_repo/builtin/packages/py_torch/package.py

```diff
@@ -538,6 +564,36 @@ def patch(self):
             "torch_global_deps PROPERTIES LINKER_LANGUAGE CXX",
             "caffe2/CMakeLists.txt",
         )
+        if self.spec.satisfies("@2.1:2.7 + rocm"):
```

Comment:
So this change has been fixed in main and the patch will no longer be needed in future releases?